daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -53,21 +67,69 @@ func Init(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 		return nil, err
 	}
 
+	opt, err := parseOptions(options)
+	if err != nil {
+		return nil, err
+	}
+
+	if userDiskQuota {
+		if err := checkFreeSpace(opt); err != nil {
+			return nil, err
+		}
+		if err := subvolEnableQuota(home); err != nil {
+			return nil, err
+		}
+		opt.quotaEnabled = true
+	}
+
 	driver := &Driver{
 		home:    home,
 		uidMaps: uidMaps,
 		gidMaps: gidMaps,
+		options: opt,
 	}
 
 	return graphdriver.NewNaiveDiffDriver(driver, uidMaps, gidMaps), nil
 }
 
+func parseOptions(opt []string) (btrfsOptions, error) {
+	var options btrfsOptions
+	options.quotaEnabled = false
```

Comment:
@cpuguy83  I need a value to determine whether to invoke func subvolEnableQuota or not. Thanks.